### ISSUE TITLE
Deleting materialization history in Lesson 8: Creating a Partition

### ISSUE THREAD

**Issue by EvanZ** [dagster-io/dagster#22996]
### What's the issue or suggestion?

In Lesson 8 there is a section where it says to delete existing materializations:

`rm $DAGSTER_HOME/storage/taxi_trips_file $DAGSTER_HOME/storage/taxi_trips $DAGSTER_HOME/storage/trips_by_week
`

However, I cannot find this path anywhere and I do not have $DAGSTER_HOME defined. I do see some oddly named folders that have been added to the repo that look like this:

`tmp1fx1r897/storage/3e76946b-2c26-450f-89f8-fdfa1687c57d`

I'm not sure what's going on here or if I missed some instruction somewhere.

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by gykung** [comment]
yes i have the same issue, stuck at this step. Using VS Code, in Windows.

**Comment by Baitur5** [comment]
you can delete using UI follow this guide https://github.com/dagster-io/dagster/discussions/18247
